pkiambo.txt

```diff
@@ -0,0 +1 @@
+Hi My name is Patsy.
```

Comment:
--- 
layout: haiku 
title: Mau Mau 
author: Patsy
---

I’m mau mau daughter<br>
For land and self governance<br>
The uprise lives on<br>